src/lib/urls.ts

```diff
@@ -4,11 +4,14 @@ export function removeDocsPath(path: string) {
     .replace('docs/', '')
     .replace('/book/', '/')
     .replace('/packages/', '/')
-    .replace('/fuels-wallet/docs/', '/fuels-wallet/')
     .replace('/apps/', '/')
     .replace('/src/', '/')
     .replace('/index.md', '.md')
     .replace('/README.md', '.md')
+    .replace('/fuel-graphql-docs/', '/graphql/')
+    .replaceAll('/fuel-specs', '/specs')
+    .replaceAll('/fuel-indexer', '/indexer')
+    .replace('/fuels-wallet/docs/', '/wallet/')
```

Comment:
We could move this to a config file to avoid the need to repeat the same map on different places.